### DISCUSSION TITLE
🟢 Checkbox | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#619] [Announcements]
# 🟢 Checkbox | Angular Challenges

Challenge 28 is about testing a simple checkbox

https://angular-challenges.vercel.app/challenges/testing/28-checkbox/

**Comment by tsironis13** [comment]
I am getting 'Project at docs has no name provided.' when trying to run 'npx nx test --watch' inside 'apps\testing\28-checkbox' folder. Any hints what is missing?

**Reply by tsironis13** [reply]
For anyone facing the same issue, the solution is to delete .nx folder and rerun the command.
